src/diffusers/pipelines/alt_diffusion/pipeline_alt_diffusion_img2img.py

```diff
@@ -81,7 +81,6 @@ class AltDiffusionImg2ImgPipeline(DiffusionPipeline):
             Model that extracts features from generated images to be used as inputs for the `safety_checker`.
     """
 
-    # Copied from diffusers.pipelines.stable_diffusion.pipeline_stable_diffusion.AltDiffusionPipeline.__init__
```

Comment:
These lines ended up here by accident before, because we have a `# Copied from` on the whole class